### PR TITLE
Update notification messages

### PR DESCRIPTION
**Pull request by classPythonAddike**
PR to implement #59 👀

### CODE REVIEW

File: TWT/apps/timathon/views/create_team.py [hunk 1]
```diff
@@ -72,12 +72,12 @@ def get(self, request: WSGIRequest) -> HttpResponse:
             client.send_webhook(
                 "Teams",
                 f"<@{context['discord_user'].uid}> tried creating a team",
-                fields=[{"name": "Error", "value": "There is not codejam ongoing"}],
+                fields=[{"name": "Error", "value": "There is no codejam ongoing"}],
```
Comment (takos22) [inline TWT/apps/timathon/views/create_team.py:75]: `There are no ongoing code-jams`

File: TWT/apps/timathon/views/create_team.py [hunk 2]
```diff
@@ -72,12 +72,12 @@ def get(self, request: WSGIRequest) -> HttpResponse:
...
             )
             return redirect("home:home")
         if challenge.team_creation_status == False:
             messages.add_message(
-                request, messages.WARNING, "Team Submissions are closed Right Now"
+                request, messages.WARNING, "Team Submissions are closed right now"
```
Comment (takos22) [inline TWT/apps/timathon/views/create_team.py:80]: `Team creation period is over`

File: TWT/apps/timathon/views/submission_list_view.py [hunk 3]
```diff
@@ -33,14 +33,14 @@ def get(self, request: WSGIRequest):
             challenge = Challenge.objects.get(ended=False, posted=True, type="MO")
         except Challenge.DoesNotExist:
             messages.add_message(
-                request, messages.INFO, "There is no ongoing challenge Right now."
+                request, messages.INFO, "There is no ongoing challenge right now."
```
Comment (takos22) [inline TWT/apps/timathon/views/submission_list_view.py:36]: `There are no ongoing challenges`

**Review by takos22 — CHANGES_REQUESTED** [review]

**Comment by classPythonAddike** [comment]
Will do the changes in an hour :cheemsdorime:

**Comment by classPythonAddike** [comment]
@takos22 committed the changes :mhm:

**Review by takos22 — APPROVED** [review]

**Pull request merged**